tests/inputFields.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from "@playwright/test";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+  await page.locator('[title="pettypes"]').click();
+  await page.waitForTimeout(3000); // wait for API response
+});
+
+test.describe('Interacting with Input Fields', async () =>{
+  test('Update pet type', async ({ page }) => {
+   
+    await expect(page.locator('h2')).toHaveText('Pet Types');
```

Comment:
Since this line was used at the beginning of every test, if can be moved to `beforeEach()` hook and removed from all 3 tests.

Reply:
done